src/components/OfferCheckoutCard/OfferCheckoutCard.js

```diff
@@ -28,34 +28,30 @@ const OfferCheckoutCard = ({ isDataLoaded, t }) => {
     freePeriods,
     freeDays,
     expiresAt,
-    startTime
+    startTime,
+    customerPriceInclTax: grossPrice
   } = useSelector(state => state.offer.offer);
-  const {
-    priceBreakdown: { offerPrice },
-    taxRate,
-    country,
-    currency
-  } = useSelector(state => state.order.order);
+  const { country, currency } = useSelector(state => state.order.order);
   const offerType = offerId?.charAt(0);
   const currencySymbol = currencyFormat[currency];
+  const formattedGrossPrice = formatNumber(grossPrice);
```

Comment:
As we will go back to using an order price value, I would move the old `grossPrice` from l:42 to l:37. Then we will be able to keep the same variable name (`grossPrice`).
Then in the translations, we will have the same variables name, which is crucial for publishers that are already using the translations feature (so in l:41 we want to keep 'grossPrice' as a variable)

Reply:
The root cause of the bug (6.99 -> 7 EUR) was caused by `formatNumber(offerPrice + taxRate * offerPrice)`
![Screen Shot 2023-03-24 at 8 33 30 AM](https://user-images.githubusercontent.com/41182484/227454552-6c4bd149-a004-4eb5-bbcf-d6aa026a2e5c.png)
This calculation is required to present properly the price for offers with trials (then, totalPrice = 0, and taxValue = 0. We need to present the final price WITH tax).
As a simple solution, I would recommend to: 
Ass util as here https://stackoverflow.com/a/11818658 and use it for `grossPrice`